folium/plugins/timestamped_geo_json.py

```diff
@@ -110,14 +114,14 @@ class TimestampedGeoJson(MacroElement):
                 })
 
             var {{this.get_name()}} = L.timeDimension.layer.geoJson(geoJsonLayer,
-                {updateTimeDimension: true,addlastPoint: {{'true' if this.add_last_point else 'false'}}}
+                {updateTimeDimension: true,addlastPoint: {{'true' if this.add_last_point else 'false'}},duration: {{"\""+this.duration+"\"" if this.duration else 'undefined'}}}
```

Comment:
Can you move the if-statement to the Python code? Where `self.duration` is set.  Makes it easier to read IMO. That way here you can just include it like `duration: "{{ this.duration }}"}`